medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -43,3 +43,8 @@ def name_search(self, name='', args=None, operator='ilike', limit=100):
 
         recs = self.search(domain + args, limit=limit)
         return recs.name_get()
+
+    @api.onchange('medical_medication_id')
+    def _onchange_medical_medication_id(self):
+        self.dispense_uom_id = \
```

Comment:
@lasley I'll check out this plugin! I remember you told me to call the method manually in my test (to test it) so I did -- but then the lint check failed on Travis, because it said that the line appeared to do nothing (so I got rid of it). Is there a better way to test it than calling `self._onchange_medical_medication_id`?